account.go

```diff
@@ -222,92 +239,62 @@ func (ea *ExternalAccount) UnmarshalJSON(b []byte) error {
 
 // LegalEntity is the structure for properties related to an account's legal state.
 type LegalEntity struct {
-	Type LegalEntityType `json:"type"`
+	Type LegalEntityType `json:"type" form:"-"` // Not updatable
 
-	AdditionalOwners []Owner `json:"additional_owners"`
+	AdditionalOwners []Owner `json:"additional_owners" form:"additional_owners,indexed"`
 
 	// AdditionalOwnersEmpty can be set to clear a legal entity's additional
 	// owners.
-	AdditionalOwnersEmpty bool
-
-	BusinessName          string               `json:"business_name"`
-	BusinessNameKana      string               `json:"business_name_kana"`
-	BusinessNameKanji     string               `json:"business_name_kanji"`
-	Address               Address              `json:"address"`
-	AddressKana           Address              `json:"address_kana"`
-	AddressKanji          Address              `json:"address_kanji"`
-	First                 string               `json:"first_name"`
-	FirstKana             string               `json:"first_name_kana"`
-	FirstKanji            string               `json:"first_name_kanji"`
-	Gender                Gender               `json:"gender"`
-	Last                  string               `json:"last_name"`
-	LastKana              string               `json:"last_name_kana"`
-	LastKanji             string               `json:"last_name_kanji"`
-	MaidenName            string               `json:"maiden_name"`
-	PersonalAddress       Address              `json:"personal_address"`
-	PersonalAddressKana   Address              `json:"personal_address_kana"`
-	PersonalAddressKanji  Address              `json:"personal_address_kanji"`
-	PhoneNumber           string               `json:"phone_number"`
-	DOB                   DOB                  `json:"dob"`
-	Verification          IdentityVerification `json:"verification"`
-	SSN                   string               `json:"ssn_last_4"`
-	SSNProvided           bool                 `json:"ssn_last_4_provided"`
-	PersonalID            string               `json:"personal_id_number"`
-	PersonalIDProvided    bool                 `json:"personal_id_number_provided"`
-	BusinessTaxID         string               `json:"business_tax_id"`
-	BusinessTaxIDProvided bool                 `json:"business_tax_id_provided"`
-	BusinessVatID         string               `json:"business_vat_id"`
+	AdditionalOwnersEmpty bool `form:"additional_owners,empty"`
```

Comment:
indentation issue

Reply:
Ah believe it or not, this is again Go's preferred styling. It's not indented with the other lines _because_ it's separated by an empty line. It's basically only groupings of lines that get aligned together.